docs/pt/orphaned/system-x-wen.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Sistema X Quando?
+description: Aprenda sobre as etapas técnicas necessárias para adicionar suporte do RetroAchievements para novos sistemas, incluindo emulação, identificação de jogos e configuração do servidor. Este artigo explica o que é necessário antes que um novo sistema possa ser suportado.
+---
+
+# Sistema X Quando?
+
+::: warning Você é um desenvolvedor de emulador?
+
+[rcheevos](https://github.com/RetroAchievements/rcheevos) é nossa biblioteca de integração de emuladores. Temos um útil [guia de integração](https://github.com/RetroAchievements/rcheevos/wiki/rc_client-integration) que pode ser útil se você deseja adicionar suporte ao RetroAchievements ao seu emulador.
+
+:::
+
+_Este artigo foi originalmente publicado no [RANews - Novembro/2020](https://news.retroachievements.org/issues/2020-11/nerd-corner)_
+
+## Introdução
+
+Há muita especulação sobre quais serão os próximos sistemas disponíveis para suportar conquistas. Com o [anúncio recente de um core de PS2](https://twitter.com/libretro/status/1322711254826471424), surgiram muitas perguntas sobre se/quando ele será suportado. Se você está interessado apenas na resposta a essa pergunta, pule para o [final deste artigo](#when-will-ra-support-playstation-2). Mas se você quer entender exatamente o que é necessário para adicionar suporte a um novo sistema, continue lendo.
+
+## Requisitos
+
+Estas são as coisas necessárias para adicionar suporte do RetroAchievements para um novo sistema:
+
+1. Emulação (preferencialmente um core libretro).
+   - O core deve expor a memória.
+   - O core deve funcionar no RALibretro.
```

Comment:
We may want to be consistent with the terminology for "core" in our pt_BR json file: https://github.com/RetroAchievements/RAWeb/blob/master/lang/pt_BR.json#L130

Reply:
Perfect, let's keep consistency! For next contributions I will refer to the pt-BR lang dict.

Reply:
Ready for re-revision! :))